SEED-platform/seed
# Bugfixes for labels (and tests)

#### What's this PR do?
Fixes some random bugs I found with labels
#### How should this be manually tested?
Ensure labels list view is working (assuming FE is in place otherwise test api call)
